irteus/irtgeo.l

```diff
@@ -425,11 +428,13 @@
                                   (or (null r)
                                       (eq r :parallel)
                                       (eq r :outside)
+                                      (and (consp r)
+                                           (eq (car r) :colinear))
                                       (and (consp r)
                                            (eq (car r) :intersect)
                                            (not (eps-in-range 0.0 (elt r 1) 1.0 (- *epsilon*)))
                                            (not (eps-in-range 0.0 (elt r 2) 1.0 (- *epsilon*)))))))
-                            (send (instance face :init :vertices (butlast rvers)) :edges))
+                            (send (instance face :init :vertices (remove v1 (cdr vers))) :edges))
```

Comment:
please add commend that why `butlast` did not work and `remove v1` works

Reply:
added https://github.com/euslisp/jskeus/pull/585/commits/5fb6cee646f0e838e32031554978685b38f527e5

`(butlast rvers)` is not the same as `rest-polygon` so the intersection is not detected correctly.
![butlast](https://user-images.githubusercontent.com/32383525/122324533-e5db5280-cf63-11eb-9497-e67e2d2acaaa.png)